packages/analytics-js/__tests__/components/configManager/commonUtil.test.ts

```diff
@@ -300,11 +300,16 @@ describe('Config Manager Common Utilities', () => {
 
       expect(state.consents.activeConsentManagerPluginName.value).toBe(undefined);
       expect(mockLogger.error).toHaveBeenCalledWith(
-        'ConfigManager:: The consent manager "randomManager" is not supported. Please choose one of the following supported consent managers: "oneTrust,ketch".',
+        'ConfigManager:: The consent manager "randomManager" is not supported. Please choose one of the following supported consent managers: "oneTrust,ketch,custom".',
       );
     });
 
     it('should log a warning if the specified storage strategy is not supported', () => {
+      state.loadOptions.value.consentManagement = {
```

Comment:
have we changes all the html files in the repo examples, public folders and sanity suite to use consentManagement instead of the previous onetrust and ketch specific options?

Reply:
Thanks for the suggestion. Updated.